build.assets/windows/build.ps1

```diff
@@ -195,45 +174,53 @@ function Get-Relcli {
         [Parameter(Mandatory)]
         [string] $Url,
         [Parameter(Mandatory)]
-        [string] $Sha256,
-        [Parameter(Mandatory)]
         [string] $Workspace
     )
     begin {
         New-Item -Path "$Workspace" -ItemType Directory -Force | Out-Null
         Invoke-WebRequest $url -UseBasicParsing -OutFile "$Workspace\relcli.exe"
-        $gotSha256 = (Get-FileHash "$Workspace\relcli.exe").hash
-        if ($gotSha256 -ne $Sha256) {
-            Write-Output "sha256 mismatch: $gotSha256 != $Sha256"
-        }
     }
 }
 
-function Register-Artifacts {
+function Generate-Artifacts {
     <#
     .SYNOPSIS
-        Invokes relcli to automatically upload built artifacts
+        Invokes relcli to automatically generate manfiests for built artifacts
     #>
     [CmdletBinding()]
     param(
         [Parameter(Mandatory)]
         [string] $Workspace,
         [Parameter(Mandatory)]
-        [string] $OutputsDir,
-        [string] $ReleaseRepo = $env:RELEASE_REPO,
-        [string] $ArtifactVersion = $env:ARTIFACT_VERSION
+        [string] $ArtifactDirectory
     )
-    begin {
-        $certPath = "$Workspace\releases.crt"
-        Out-File -FilePath $certPath -Encoding ascii -InputObject "$env:RELEASES_CERT"
-        $keyPath = "$Workspace\releases.key"
-        Out-File -FilePath $keyPath -Encoding ascii -InputObject "$env:RELEASES_KEY"
 
-        # These must be set for the `auto_upload` command
-        $env:DRONE_REPO = "$ReleaseRepo"
-        $env:DRONE_TAG = "$ArtifactVersion"
+    $SearchPath = Join-Path -Path $ArtifactDirectory -ChildPath *
+    Get-ChildItem -Path $SearchPath -Include "*.exe","*.zip" | ForEach-Object {
+        switch -Wildcard ($_.Name) {
+            "Teleport Connect Setup*.exe" {
+                $description = "Teleport Connect"
+                Break
+            }
+            "teleport-windows-auth-setup*.exe" {
+                $description = "Teleport Connect"
```

Comment:
Should this have a different description?
Same for the other archive in the next switch case